stage1/init/common/pod.go

```diff
@@ -348,6 +380,18 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		opts = append(opts, unit.NewUnitOption("Service", "SyslogIdentifier", filepath.Base(app.Exec[0])))
 	}
 
+	if flavor == "kvm" {
+		capabilities := kvm.GetAppCapabilities(app)
+		capabilities = append(capabilities, stdCapabilities...)
```

Comment:
`nspawn` is starting with pre-defined capabilities (as an isolation from host). Kvm is starting whole VM with **full** `systemd` (which has got every capability and You cannot(and really, You won't ;)) change them). This patch delivers standard capabilities for applications inside pod (so Your pod environment has got every single capability, but Your application don't).